app/src/main/java/com/simplecity/amp_library/saf/SafManager.java

```diff
@@ -285,6 +285,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
                     .title(R.string.saf_access_required_title)
                     .content(R.string.saf_access_required_message)
                     .positiveText(R.string.saf_show_files_button)
+                    .positiveColor(getResources().getColor(R.color.colorPrimaryDark))
```

Comment:
Please using `ShuttleApplication.get().getResources()` instead for all of these resources to make sure that you can always access the resource. Otherwise you can end up with an NPE if the local Activity context dies.